eZ/Publish/Core/REST/Common/FieldTypeProcessor/BaseRelationProcessor.php

```diff
@@ -16,21 +17,55 @@
      */
     private $router;
 
+    /**
+     * @var \eZ\Publish\Core\Repository\LocationService
+     */
+    private $locationService;
+
+    /**
+     * @param \Symfony\Component\Routing\RouterInterface $router
+     */
     public function setRouter(RouterInterface $router)
     {
         $this->router = $router;
     }
 
+    /**
+     * @param \eZ\Publish\API\Repository\LocationService $locationService
+     */
+    public function setLocationService(LocationService $locationService)
+    {
+        $this->locationService = $locationService;
+    }
+
+    /**
+     * @return bool
+     */
     public function canMapContentHref()
     {
         return isset($this->router);
     }
 
+    /**
+     * @param  int $contentId
+     * @return string
+     */
     public function mapToContentHref($contentId)
     {
         return $this->router->generate('ezpublish_rest_loadContent', ['contentId' => $contentId]);
     }
 
+    /**
+     * @param  int $locationId
+     * @return string
+     */
+    public function mapToLocationHref($locationId)
+    {
+        return $this->router->generate('ezpublish_rest_loadLocation', [
```

Comment:
it seems this generates `http://ez.loc/api/ezp/v2/content/locations//1/43/` (notice the double `/`) while this should be `http://ez.loc/api/ezp/v2/content/locations/1/43/` (with the `//` the REST API answers with a 404)